compiler/damlc/tests/src/DamlcTest.hs

```diff
@@ -178,7 +198,7 @@ testsForDamlcTest damlc = testGroup "damlc test" $
               , "x = scenario $ assert False"
               ]
             (exitCode, stdout, stderr) <- readProcessWithExitCode damlc ["test", "--files", file] ""
-            stdout @?= ""
+            stdout @?= "test coverage: templates 100%, choices 100%\n"
```

Comment:
I’d hide the test coverage for failed invocations. It isn’t correct (you’re excluding coverage from the failed tests), and it just doesn’t seem very useful. fix your tests first, then look at coverage.

Reply:
now not printed anymore on failure.